Can't install with Node.js 26

The current version of `nan` used by `node-libpq` is incompatible with Node.js 26:
```
➜  node-libpq git:(master) nvm use 26
Now using node v26.0.0 (npm v11.12.1)
➜  node-libpq git:(master) npm install

> libpq@1.10.0 install
> node-gyp rebuild

gyp info it worked if it ends with ok
gyp info using node-gyp@12.2.0
gyp info using node@26.0.0 | darwin | arm64
gyp info find Python using Python version 3.9.6 found at "/Library/Developer/CommandLineTools/usr/bin/python3"

# Lots of warnings snipped for brevity's sake

In file included from ../src/connection.cc:1:
In file included from ../src/addon.h:4:
../node_modules/nan/nan.h:2596:9: error: no type named 'AccessControl' in namespace 'v8'
 2596 |   , v8::AccessControl settings
      |     ~~~~^
../node_modules/nan/nan.h:2653:9: error: no type named 'AccessControl' in namespace 'v8'
 2653 |   , v8::AccessControl settings = v8::DEFAULT
      |     ~~~~^
../node_modules/nan/nan.h:2653:34: error: no member named 'DEFAULT' in namespace 'v8'; did you mean 'UV_EFAULT'?
 2653 |   , v8::AccessControl settings = v8::DEFAULT
      |                                  ^~~~~~~~~~~
      |                                  UV_EFAULT
/Users/allon/Library/Caches/node-gyp/26.0.0/include/node/uv.h:193:16: note: 'UV_EFAULT' declared here
  193 |   UV_ERRNO_MAP(XX)
      |                ^
In file included from ../src/connection.cc:1:
In file included from ../src/addon.h:4:
../node_modules/nan/nan.h:2706:9: error: no type named 'AccessControl' in namespace 'v8'
 2706 |   , v8::AccessControl settings = v8::DEFAULT
      |     ~~~~^
../node_modules/nan/nan.h:2706:34: error: no member named 'DEFAULT' in namespace 'v8'; did you mean 'UV_EFAULT'?
 2706 |   , v8::AccessControl settings = v8::DEFAULT
      |                                  ^~~~~~~~~~~
      |                                  UV_EFAULT
```

--- a/.github/workflows/ci.yaml
+++ b/.github/workflows/ci.yaml
@@ -22,7 +22,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [18.x, 20.x, 22.x, 24.x]
+        node-version: [18.x, 20.x, 22.x, 24.x, 26.x]
 
     steps:
       - uses: actions/checkout@v3
@@ -30,6 +30,11 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
+      - name: Override yargs for Node.js 26
+        if: ${{ matrix.node-version == '26.x' }}
+        run: |
+          npm pkg set overrides.yargs=18.0.0
+          npm install
       - run: npm ci
       - run: npm run build --if-present
       - run: PGTESTNOSSL=true PGUSER=postgres PGHOST=localhost PGPASSWORD=postgres PGDATABASE=ci_db_test npm test

--- a/package.json
+++ b/package.json
@@ -20,7 +20,7 @@
   "license": "MIT",
   "dependencies": {
     "bindings": "1.5.0",
-    "nan": "~2.23.1"
+    "nan": "~2.26.2"
   },
   "devDependencies": {
     "async": "^2.6.2",

